com.ibm.streamsx.metrics/impl/java/src/com/ibm/streamsx/metrics/internal/TupleContainer.java

```diff
@@ -227,6 +232,17 @@ public void setJobName(String jobName) {
 			_tuple.setString(_jobNameAttributeIndex, jobName);
 		}
 	}
+	
+	/**
+	 * Optionally set the pe id in the output tuple.
+	 * 
+	 * @param peId
+	 */
+	public void setPeId(BigInteger peId) {
```

Comment:
You shouldn't be able to compile because the same method exists in line 292 in the master.